virttest/utils_libvirt/libvirt_memory.py

```diff
@@ -0,0 +1,20 @@
+"""
+Virtualization test - utility functions for libvirt
+
+:copyright: 2021 Red Hat Inc.
```

Comment:
Could you please add 'memory' to make this file more specific? 
BTW, please update '2021' to '2022'. Thanks!

Reply:
Updated, thanks.